docs/operating-scylla/nodetool-commands/disableautocompaction.rst

```diff
@@ -24,10 +24,11 @@ nodetool disableautocompaction takes the following parameters:
 
    * - Parameter Name
      - Description
-   * - ``[<keyspace> <tables>...]``
-     - The keyspace followed by one or many tables      
-            
+   * - ``[<keyspace>]``
+     - The keyspace to operate on.  If omitted, auto-compaction will be disabled in all keyspaces.
```

Comment:
Hmm, isn't this dangerous? It also disables compaction on system tables? Do we even want to allow a user to do this by mistake?

Reply:
Our API requires a keyspace name, but, today, nodetool looks up all keyspaces if this parameter is omitted.

Reply:
See https://github.com/scylladb/scylla-tools-java/blob/86a200e324fac3b4a47b7954e30de6e05a4e7ce0/src/java/org/apache/cassandra/tools/nodetool/DisableAutoCompaction.java#L39
and https://github.com/scylladb/scylla-tools-java/blob/86a200e324fac3b4a47b7954e30de6e05a4e7ce0/src/java/org/apache/cassandra/tools/NodeTool.java#L386-L403

Reply:
I just realized I missed this case in my scylla-native snapshot implementation.

Reply:
I'm not sure what was the original use case for this.  It's used naively by dtest (that don't care about system keyspaces), and I guess it could have been used prior to taking a snapshot and/or drain/shutdown, but it is not required anymore for those use cases.